ci/integration.js

```diff
@@ -0,0 +1,236 @@
+const child_process = require("child_process");
+const { randomBytes } = require("crypto");
+const path = require("path");
+const fs = require("fs");
+
+global.XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
+
+const { ajax } = require("rxjs/ajax");
+
+const rxJupyter = require("rx-jupyter");
```

Comment:
`rx-jupyter` isn't actually used here, need https://github.com/nteract/nteract/pull/3651 to be merged and shipped.